src/_base/harness/scripts/xdebug_version_sync.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+PREPARE_CONFIG=0
+RESTART_FPM=0
+
+xdebug_version_sync()
+{
+  INSTALLED_VERSION=$(docker-compose exec -T -u root "$1" pecl info xdebug 2>&1 | grep "Release Version" | awk '{print $3}')
+  INSTALLED_MAJOR_VERSION=$(echo "$INSTALLED_VERSION" | cut -d '.' -f1)
+
+  if [ "$INSTALLED_MAJOR_VERSION" == "$XDEBUG_VERSION" ]; then
```

Comment:
```suggestion
  if [[ "$INSTALLED_MAJOR_VERSION" == "$XDEBUG_VERSION" ]]; then
```
as `==` is only available in double brackets

Reply:
I've added the double brackets around the check for restarting the container too since I was using `==` there also.